app/controllers/veterans/registrations_controller.rb

```diff
@@ -11,7 +11,7 @@ def create
       if @veteran.save
         send_notifications
         ab_finished(:signup_button_size)
-        redirect_to '/', notice: 'Thanks for signing up!'
+        redirect_to '/veterans/thanks', notice: 'Thanks for signing up!'
```

Comment:
I think we're safe removing the flash since the user is thanked on the page they are directed to.